app/presenters/hyrax/batch_ingest/batch_item_presenter.rb

```diff
@@ -56,15 +65,17 @@ def status_css_classes
             running:      'fa fa-refresh fa-sync',
             completed:    'fa fa-check-circle',
             failed:       'fa fa-exclamation-triangle',
-            expunged:     'fa fa-exclamation-triangle'
-          }.with_indifferent_access
+            # NOTE: gone is not a value for the status field, but meant to be
+            # used when repo_object_gone?
+            gone:         'fa fa-exclamation-circle'
```

Comment:
Is it possible to remove this method `status_css_classes`?  I don't see it being used anymore after the refactor above.